bot/seasons/halloween/hacktoberstats.py

```diff
@@ -10,6 +10,8 @@
 import discord
 from discord.ext import commands
 
+from bot.utils.persist import datafile
```

Comment:
I thought I had added this to the previous review, but apparently not:

```suggestion
from bot.utils.persist import make_persistent
```